go/host/db/blocks.go

```diff
@@ -13,16 +12,7 @@ import (
 	"github.com/ethereum/go-ethereum/rlp"
 )
 
-// DB methods relating to batches.
-
-// GetHeadBlockHeader returns the block header of the current head block.
-func (db *DB) GetHeadBlockHeader() (*types.Header, error) {
```

Comment:
Storing and retrieving the head block is no longer required - we only did it to power that test endpoint.